Docs/MigrationToNodeJsExtensions.md

```diff
@@ -0,0 +1,71 @@
+# Migrating to Lombiq Node.js Extensions
+
+## 1. Update your project file
```

Comment:
Is it necessary to number the section titles? Looks a bit weird imo.

Reply:
Well, I wanted it to be a list, but that would increase indentation. It is "weird" in a sense, I agree, and if you find if offending enough, I can remove them.

On the other hand, the order is important. Do you think people won't just do the steps in any order if we remove the numbers?

Reply:
I think anyone who does this out of order is surely insane and can't be reasoned with! :D
You could add a preamble like "Please do these steps in order." to line 2.